types/cesium/index.d.ts

```diff
@@ -2119,41 +2120,43 @@ declare namespace Cesium {
 
     class BillboardGraphics {
         definitionChanged: Event;
-        image: Property;
-        imageSubRegion: Property;
-        scale: Property;
-        rotation: Property;
-        alignedAxis: Property;
-        horizontalOrigin: Property;
-        verticalOrigin: Property;
-        color: Property;
-        eyeOffset: Property;
-        pixelOffset: Property;
-        show: Property;
-        width: Property;
-        height: Property;
-        scaleByDistance: Property;
-        translucencyByDistance: Property;
-        disableDepthTestDistance: Property;
-        pixelOffsetScaleByDistance: Property;
-        constructor(options?: { image?: Property;
-            show?: Property;
-            scale?: Property;
-            horizontalOrigin?: Property;
-            verticalOrigin?: Property;
-            eyeOffset?: Property;
-            pixelOffset?: Property;
-            rotation?: Property;
-            alignedAxis?: Property;
-            width?: Property;
-            height?: Property;
-            color?: Property;
-            scaleByDistance?: Property;
-            translucencyByDistance?: Property;
-            pixelOffsetScaleByDistance?: Property;
-            disableDepthTestDistance?: Property;
-            imageSubRegion?: Property
-            heightReference?: Property;
+        image: Property | string;
+        imageSubRegion: Property | BoundingRectangle;
+        scale: Property | number;
+        rotation: Property | number;
+        alignedAxis: Property | Cartesian3;
+        horizontalOrigin: Property | HorizontalOrigin;
+        verticalOrigin: Property | VerticalOrigin;
+        color: Property | Color;
+        eyeOffset: Property | Cartesian3;
+        pixelOffset: Property | Cartesian2;
+        show: Property | boolean;
+        width: Property | number;
+        height: Property | number;
+        heightReference: Property | HeightReference;
+        scaleByDistance: Property | NearFarScalar;
+        translucencyByDistance: Property | NearFarScalar;
+        disableDepthTestDistance: Property | number;
+        pixelOffsetScaleByDistance: Property | NearFarScalar;
```

Comment:
The constructor options for `Graphics` objects can be either a Property or the raw value, but the properties on the objects cannot be, even though the library technically supports it. See my comment here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44143/files#r413747367

Cesium will always wrap whatever you set here in a `Property`, so unless Typescript allows us to someday define different set/get types for properties like this, we have to use the more restrictive type. Otherwise the types on reads are incorrect and we lose typesafety.

Reply:
So, what can i do?  Types Property suck my energy, because my vscode tell me - lengh: Property.. and i must open documentation, where i see - its a number.. so, Property | number is good compromise. 

If you tell me, what can i do, i try to refactor it